src/Table/Table.tsx

```diff
@@ -51,7 +52,7 @@ export interface ActionLinkItem {
 }
 
 export interface PrimaryActionButton {
-    label: string;
+    label: string | React.ReactNode;
```

Comment:
I believe `ReactNode` interface extends `ReactText` which is a `string` or `number`
```suggestion
    label: React.ReactNode;
```